components/Base/VsBrMenu.vue

```
@@ -400,6 +398,12 @@ import VsBrDivider from '~/components/Modules/VsBrDivider.vue';
 import VsBrMegaNav from '~/components/Modules/VsBrMegaNav.vue';
 import VsBrAccordionNav from '~/components/Modules/VsBrAccordionNav.vue';
 
+import { getCurrentInstance } from 'vue'
```

ESLint

[error] '/home/jailuser/git/node_modules/vue/dist/vue.runtime.esm-bundler.js' imported multiple times.

(import/no-duplicates)